mypy/test/testpep561.py

```diff
@@ -102,127 +175,162 @@ def install_package(self, pkg: str,
             self.fail('\n'.join(lines))
 
     def setUp(self) -> None:
-        self.temp_file_dir = tempfile.TemporaryDirectory()
-        self.tempfile = os.path.join(self.temp_file_dir.name, 'simple.py')
-        with open(self.tempfile, 'w+') as file:
-            file.write(SIMPLE_PROGRAM)
-        self.namespace_tempfile = os.path.join(self.temp_file_dir.name, 'namespace_program.py')
-        with open(self.namespace_tempfile, 'w+') as file:
-            file.write(NAMESPACE_PROGRAM)
-
-        self.msg_dne = \
-            "{}:3: error: Module 'typedpkg' has no attribute 'dne'\n".format(self.tempfile)
-        self.msg_list = \
-            "{}:5: error: Revealed type is 'builtins.list[builtins.str]'\n".format(self.tempfile)
-        self.msg_tuple = \
-            "{}:5: error: Revealed type is 'builtins.tuple[builtins.str]'\n".format(self.tempfile)
-
-        self.namespace_msg_bool_str = (
-            '{0}:8: error: Argument 1 to "nested_func" has incompatible type "bool"; '
-            'expected "str"\n'.format(self.namespace_tempfile))
-        self.namespace_msg_int_bool = (
-            '{0}:9: error: Argument 1 to "alpha_func" has incompatible type "int"; '
-            'expected "bool"\n'.format(self.namespace_tempfile))
+        self.simple_example_program = ExampleProgram(SIMPLE_PROGRAM)
```

Comment:
These variable names are all too long -- when skimming all I see is `blah_blah_blah_blah`. I don't think they all need to end in `_example_program`, and you can abbreviate `namespace` as `ns`.